packages/react-composites/src/composites/CallWithChatComposite/components/PreparedDialpad.tsx

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+import { useState } from 'react';
+import { useMemo } from 'react';
+import { Dialpad, DialpadStyles } from '@internal/react-components';
+import { _DrawerMenu, _DrawerMenuItemProps, _DrawerSurface } from '@internal/react-components';
+import { IModalStyles, Modal, Stack, useTheme, Text, IconButton } from '@fluentui/react';
+import { IButtonStyles, PrimaryButton } from '@fluentui/react';
+import { drawerContainerStyles } from '../styles/CallWithChatCompositeStyles';
+import { themedDialpadModelStyle } from './PreparedDialpad.styles';
+import { themedCallButtonStyle, themedDialpadStyle } from './PreparedDialpad.styles';
+/* @conditional-compile-remove(PSTN-calls) */
+import { CallWithChatCompositeIcon } from '../../common/icons';
+
+/** @private */
+export interface PreparedDialpadStrings {
+  dialpadModalAriaLabel: string;
+  dialpadModalTitle: string;
+  dialpadCloseModalButtonAriaLabel: string;
+  dialpadStartCallButtonLabel: string;
+}
+
+/** @private */
+export interface PreparedDialpadProps {
+  isMobile: boolean;
+  showDialpad: boolean;
+  strings: PreparedDialpadStrings;
+  onDismissDialpad: () => void;
+}
+
+/** @private */
+export const PreparedDialpad = (props: PreparedDialpadProps): JSX.Element => {
```

Comment:
Why is it called a PreparedDialpad?

Reply:
It's a dialpad with call button implemented and all styling required for being in the composite :)

Reply:
Given that we will have (most probably) another dialpad in the composites for just sending DTMF tones without the start call button, any idea what that dialpad would be called?

Since in this context that dialpad would also be a "prepared" dialpad for sending dtmf tones.

Reply:
PreparedDialpadForDTMF? and this one we can call it PreparedDialpadForCalling

Reply:
I would shorten it to `CallingDialpad` and `DTMFDialpad` to keep the naming crisp.

Reply:
Personally `Prepared` as a prefix is not adding value here since these components exists in the composites package already.